reference/configuration/swiftmailer.rst

```diff
@@ -156,6 +157,16 @@ emails sent during development go to a single account.
 This uses ``Swift_Plugins_RedirectingPlugin``. Original recipients are available
 on the ``X-Swift-To``, ``X-Swift-Cc`` and ``X-Swift-Bcc`` headers.
 
+delivery_whitelist
+~~~~~~~~~~~~~~~~~~
+
+**type**: ``array``
+
+Used in combination with ``delivery_address``. If set, send email as normal to 
+addresses that match with the values in the list. All addresses not matching will 
+be sent to ``delivery_address`` as is the normal behavior when setting a 
+``delivery_address``.
```

Comment:
We should include a link to the now-merged new section about this: http://symfony.com/doc/current/cookbook/email/dev_environment.html#sending-to-a-specified-address-but-with-exceptions

And with this link, we can make the description here ever shorter - perhaps just:

```rst
Used in combination with ``deliver_address``. If set, emails matching this pattern will
be delivered like normal, instead of being sent to `delivery_address`. For details, see
LINK...
```

We'll need to add a new anchor link above that section (e.g. `.. _cookbook-emails-delivery_whitelist`) so we can link to it.

Thanks!